### ISSUE TITLE
Unable to navigate to any tab except Speakers when VoiceOver accessibility option enabled on iOS

### ISSUE THREAD

**Issue by sebj** [paug/AndroidMakersApp#275]
When the VoiceOver accessibility option is enabled on iOS, I'm unable to navigate to any of the tabs except for Speakers. 

The entire tab bar also seems to appear as a single accessibility element with no description, compared to native iOS apps which will usually have a selectable element for each tab, with a description in the format "<Tab Name>, Tab, <Tab Number> of <Total Tab Count>". Double-tapping to perform an action on the tab bar always selects the Speakers tab, regardless of what other tab I'm on.

Note in the screenshot below the black bounding box for the selectable element, and the description for Android Makers which is of the last describable element on screen as the tab bar seemingly has none.

| App Store | Header |
|--------|--------|
| ![App Store with VoiceOver](https://github.com/paug/AndroidMakersApp/assets/379991/3fe867e6-3eff-4550-95d7-782a7d24b5c8) | ![Android Makers with VoiceOver](https://github.com/paug/AndroidMakersApp/assets/379991/a3c32d00-4a1f-44aa-931a-7b0add8ac08d) |

**Comment by cbeyls** [comment]
Need to monitor if this was fixed in recent Compose Multiplatform versions
